src/main/java/com/feedzai/commons/sql/abstraction/engine/impl/DB2Engine.java

```diff
@@ -124,6 +125,11 @@ protected void setPreparedStatementValue(final PreparedStatement ps,
             case JSON:
             case CLOB:
             case BLOB:
+                if (isNull(value)) {
+                    ps.setBytes(index, null);
+                    break;
+                }
```

Comment:
This looks ok to me, but since it's exactly the same in all implementations, it might as well be in AbstractDatabaseEngine.  
However, the same could be said about a lot more code, so...